server/src/main/java/server/cache/BUILD

```diff
@@ -0,0 +1,18 @@
+load("@rules_java//java:defs.bzl", "java_library")
+
+package(default_visibility = ["//visibility:public"])
+
+java_library(
+    name = "cache",
```

Comment:
See comments below, but perhaps we ought to move all this code into the `server.bazel` package?